### ISSUE TITLE
Should check `.log(level, msg)` calls too

### ISSUE THREAD

**Issue by andersk** [globality-corp/flake8-logging-format#69]
```python
import logging
logger = logging.getLogger("test")
a, b = "a", "b"
logger.error(f"{a} {b}")  # raises G004 Logging statement uses f-string
logger.log(logging.ERROR, f"{a} {b}")  # should raise G004, but doesn’t
```
